azext_hanaonazure/custom.py

```diff
@@ -28,3 +28,17 @@ def restart_hanainstance(client, resource_group_name, instance_name):
 
 def update_hanainstance(client, resource_group_name, instance_name, **kwargs):
     return client.update(resource_group_name, instance_name, kwargs['parameters'].tags)
+
+def enable_monitoring_hanainstance(client, resource_group_name, instance_name, hana_vnet, hana_hostname, hana_instance_num, hana_db_username, hana_db_password, db_container="single", hana_database=None):
+    monitoring_details = {
+        "hanaInstanceNum": hana_instance_num,
+        "hanaVnet": hana_vnet,
+        "hanaHostname": hana_hostname,
+        "hanaDbUsername": hana_db_username,
+        "hanaDbPassword": hana_db_password,
+        "dbContainer": db_container
```

Comment:
Again, not a fan of `dbContainer` vs. a boolean `HanaMdc`:
`dbContainer` seems to refer more to the container (aka the tenant DB itself).

Reply:
This is the one place where I can actually change this in this PR.